src/components/Navbar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { FaArrowLeft, FaSearch } from 'react-icons/fa';
+import { HiCog } from 'react-icons/hi';
+import '../styles/Navbar.css';
+
+function Navbar() {
+  return (
+    <nav>
+      <div className="header">
+        <ul className="back-stats">
+          <li><NavLink to="/"><FaArrowLeft /></NavLink></li>
+          <li>WorldWide Covid Statistics</li>
+          <li className="mic-settings">
+            <div><FaSearch /></div>
```

Comment:
As specified in the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/be3b13a85ca425c0bf0eeb1f130b6a52e36deb4f/capstone/react_capstone.md?plain=1#L67), kindly include some form of filtering by categories or a search (you already have a search icon).